Sources/PackageGraph/ModuleAliasTracker.swift

```diff
@@ -72,7 +72,7 @@ class ModuleAliasTracker {
     func trackTargetsPerProduct(product: Product,
                                 package: PackageIdentity) {
         let targetDeps = product.targets.map{$0.dependencies}.flatMap{$0}
-        var allTargetDeps = product.targets.map{$0.dependentTargets().map{$0.dependencies}}.flatMap{$0}.flatMap{$0}
+        var allTargetDeps = product.targets.map{$0.recurisveDependentTargets().map{$0.dependencies}}.flatMap{$0}.flatMap{$0}
```

Comment:
Small typo here in `recursiveDependentTargets`, not really super important for 5.7, but we should probably update in `main`.

Reply:
oops, will update